### PR TITLE
Settings Page: css changes

### PR DESCRIPTION
**Pull request by W3stside**
Simple css changes on `/settings` page

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr1185--dexreact.review.gnosisdev.com

File: src/components/Settings/WalletConnect.tsx [hunk 1]
```diff
@@ -131,54 +134,87 @@ const FormField = styled.label`
...
+    &:focus {
+      border-color: var(--color-background-balance-button-hover);
+    }
   }
 `
 
 export const WCSettings: React.FC<WCSettingsProps> = ({ register, errors }) => {
   return (
     <div>
+      <h1>Connection Settings</h1>
```
Comment (Velenir) [inline src/components/Settings/WalletConnect.tsx:201]: Maybe emphasize that these are **WalletConnect** Connection settings

**Comment by anxolin** [comment]
I don't think it's super important. My concern is more why we always have to styles individual pages differently. I see a lot of custom CSS code, and I think it doesn't help to get a consistent UI across pages. 

What's in your opinion the best way to style the web so our texts, inputs, labels, are more consistent?

**Comment by Velenir** [comment]
> What's in your opinion the best way to style the web so our texts, inputs, labels, are more consistent?

Style system like a styleguidist, enforced through storybook
Too much hassle of course

**Pull request merged**